frontend/app/.server/utils/locale.utils.ts

```diff
@@ -2,14 +2,87 @@ import type { Params } from '@remix-run/react';
 
 import type { Namespace } from 'i18next';
 import { createInstance } from 'i18next';
-import I18NexFsBackend from 'i18next-fs-backend';
-import { resolve } from 'node:path';
 import { initReactI18next } from 'react-i18next';
 
+import addressValidationEn from '~/../public/locales/en/address-validation.json';
+import applyAdultChildEn from '~/../public/locales/en/apply-adult-child.json';
+import applyAdultEn from '~/../public/locales/en/apply-adult.json';
+import applyChildEn from '~/../public/locales/en/apply-child.json';
+import applyEn from '~/../public/locales/en/apply.json';
+import dataUnavailableEn from '~/../public/locales/en/data-unavailable.json';
+import gcwebEn from '~/../public/locales/en/gcweb.json';
+import indexEn from '~/../public/locales/en/index.json';
+import lettersEn from '~/../public/locales/en/letters.json';
+import protectedRenewEn from '~/../public/locales/en/protected-renew.json';
+import renewAdultChildEn from '~/../public/locales/en/renew-adult-child.json';
+import renewChildEn from '~/../public/locales/en/renew-child.json';
+import renewItaEn from '~/../public/locales/en/renew-ita.json';
+import renewEn from '~/../public/locales/en/renew.json';
+import statusEn from '~/../public/locales/en/status.json';
+import stubLoginEn from '~/../public/locales/en/stub-login.json';
+import unableToProcessRequestEn from '~/../public/locales/en/unable-to-process-request.json';
+import addressValidationFr from '~/../public/locales/fr/address-validation.json';
+import applyAdultChildFr from '~/../public/locales/fr/apply-adult-child.json';
+import applyAdultFr from '~/../public/locales/fr/apply-adult.json';
+import applyChildFr from '~/../public/locales/fr/apply-child.json';
+import applyFr from '~/../public/locales/fr/apply.json';
+import dataUnavailableFr from '~/../public/locales/fr/data-unavailable.json';
+import gcwebFr from '~/../public/locales/fr/gcweb.json';
+import indexFr from '~/../public/locales/fr/index.json';
+import lettersFr from '~/../public/locales/fr/letters.json';
+import protectedRenewFr from '~/../public/locales/fr/protected-renew.json';
+import renewAdultChildFr from '~/../public/locales/fr/renew-adult-child.json';
+import renewChildFr from '~/../public/locales/fr/renew-child.json';
+import renewItaFr from '~/../public/locales/fr/renew-ita.json';
+import renewFr from '~/../public/locales/fr/renew.json';
+import statusFr from '~/../public/locales/fr/status.json';
+import stubLoginFr from '~/../public/locales/fr/stub-login.json';
+import unableToProcessRequestFr from '~/../public/locales/fr/unable-to-process-request.json';
 import { getEnv } from '~/.server/utils/env.utils';
 import { getLogger } from '~/.server/utils/logging.utils';
 import { APP_LOCALES } from '~/utils/locale-utils';
 
+export const i18nResources = {
+  en: {
+    addressValidation: addressValidationEn,
```

Comment:
Uses `kebab-case` to match Typescript keys augmentation.

```suggestion
    "address-validation": addressValidationEn,
```

Reply:
Oops.. regex replace error. I'll fix.
I tested the types but of course I chose `gcweb` and `index` to test with.. 🙄

Reply:
Put emphasis on `en`. We can use tools like [i18next-locales-sync](https://github.com/felixmosh/i18next-locales-sync) to synchronize locales with `en`.

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/pull/2747/commits/842414d108676f00817521249680e04f0dbbeef7)